keras_cv/visualization/plot_image_gallery.py

```diff
@@ -80,6 +73,10 @@ def unpackage_tfds_inputs(inputs):
         legend_handles: (Optional) matplotlib.patches List of legend handles.
             I.e. passing: `[patches.Patch(color='red', label='mylabel')]` will
             produce a legend with a single red patch and the label 'mylabel'.
+
+    Note:
```

Comment:
Instead of assuming the dataset has a dict with the key "image", how about the elements are just the images?

Reply:
Normally, TFDS and most other datasets which are pre-loaded have a dictionary. Do you think it is unnecessary?

Reply:
okay, for compat with tfds that seems reasonable. Let's move this note to the `images` docstring.

At the end of the docstring we can say "If a `tf.data.Dataset` is used, images should be present the `FeaturesDict` under the key `"image"`